test/s3-tests/src/test/resources/log4j2.xml

```diff
@@ -24,6 +24,7 @@
         <Root level="WARN">
             <AppenderRef ref="ConsoleAppender"/>
         </Root>
+        <Logger name="software.amazon.awssdk.services.s3.regression" level="DEBUG" />
```

Comment:
nit: maybe just changing root level to INFO and making those logs in this class INFO so that we don't end up printing tons of logs?